config/docker/docker-compose.dev.yml

```diff
@@ -43,6 +43,7 @@ services:
       - /tmp
     networks:
       - noticed-flight-sim-net
+    command: bash -c "bin/rails s -p 3000 -b '0.0.0.0'"
```

Comment:
Found this handy but I could see it be annoying too, if you want to restart the server.